client/src/components/SignInForm/state/handleInputBlur.js

```diff
@@ -1,3 +1,26 @@
-const handleInputBlur = (state) => state;
+import { formValuesFromState } from 'models/form';
```

Comment:
I like the idea of moving these handlers out of your component's file. It cleans things up a bit